app/src/App.vue

```diff
@@ -0,0 +1,199 @@
+                                            <div class="product-suggestion-name">
+                                                <div>
+                                                    <span class="product-suggestion-attributes">{{ item.name }}</span><br>
+                                                    <span v-if="item.attribute_name" class="product-suggestion-attributes">{{ item.attribute_name }}</span>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </template>
+                                </el-autocomplete>
+                            </el-form-item>
+
+                            <el-form-item v-if="form.selectedProduct">
+                                <div>
+                                    <div class="selected-product-image">
+                                        <img :src="form.selectedProduct.image_link" class="product-suggestion-image" width="75" height="75">
+                                    </div>
+                                    <div class="selected-product-name">
+                                        <div>
+                                            <label>{{ form.selectedProduct.name }}</label><br>
+                                            <label>{{ form.selectedProduct.attribute_name }}</label>
+                                        </div>
+                                        <div class="selected-product-delete">
+                                            <i class="material-icons red" @click="removeSelectedProduct()">close</i>
+                                        </div>
+                                    </div>
+                                </div>
+                            </el-form-item>
+
+                            <el-form-item :label="translations.action" prop="selectedAction">
+                                <el-select v-model="form.selectedAction" :placeholder="translations.action" clearable>
+                                    <el-option label="Go to cart" value="1"></el-option>
+                                    <el-option label="Check out" value="2"></el-option>
+                                </el-select>
+                            </el-form-item>
+
+                            <el-form-item :label="translations.sharableLink" class="form-link">
+                                <span v-if="form.selectedProduct && form.selectedAction" class="generated-link">{{ ps_base_url }}/index.php?fc=module&module=ps_buybuttonlite&controller=RedirectManager&id_product={{ form.selectedProduct.id_product }}&action={{ form.selectedAction }}<label v-if="form.selectedProduct.id_product_attribute">&id_product_attribute={{ form.selectedProduct.id_product_attribute }}</label></span>
+                                <span v-else class="no-link">Please select a product and an action</span>
```

Comment:
Is it a translation key?

Reply:
Indeed this does not look translatable
=> how do you plan to handle translations in Vue.js ?

Reply:
Oops forgot that one. Solved.